packages/remote-config/lib/modular/index.js

```diff
@@ -261,3 +261,5 @@ export async function setCustomSignals(remoteConfig, customSignals) {
   }
   return remoteConfig._promiseWithConstants(remoteConfig.native.setCustomSignals(customSignals));
 }
+
+export { LastFetchStatus, ValueSource } from '../statics';
```

Comment:
there is a slight difference in the statics export here (modular only, but used in namespaced) vs the storage one (exported in both modular and namespaced, but looks like storage statics export. Unsure which should be The Way

perhaps in storage we don't need the export in namespaced index.js ?

Reply:
We should only export these from modular in my view. namespaced statics should be from namespace only (e,g, RemoteConfig.Statics.ValueSource).

Reply:
agreed - I like that style